### DISCUSSION TITLE
Building EdgeTX

### DISCUSSION THREAD

**Discussion by K2AAE** [EdgeTX/edgetx#1728] [General]
I setup a build in WSLq / Ubuntu 20.04 and built a release but it is 2.4 where is a guide on getting the current build? I couldn't find anything about adding features to EdgeTX anywhere.

**Comment by piotrva** [comment]
As you check out main branch from git it should get you a built of 2.7 - please check what commit have you checked out from git.
And I suppose discord server would be better place to get such support ;)

**Comment by ghost** [comment]
cmake -DPCB=X10 -DPCBREV=EXPRESS -DHELI=NO -DLUA_MIXER=YES -DCMAKE_BUILD_TYPE=Release ../

That is the command line, where you add or remove features.

Building for my X10 Express with HELI removed and LUA MIXER added.
